conf/messages

```diff
@@ -414,46 +414,16 @@ checkYourAnswers.reasonableExcuse = Reason for missing the VAT deadline
 checkYourAnswers.change.text = Change
 checkYourAnswers.change.ariaLabel = Change answer for ’{0}’
 checkYourAnswers.whyYouDidNotAppealSooner = Reason for appealing after 30 days
-checkYourAnswers.fireOrFlood.whenDidTheFireOrFloodHappen = When did the fire or flood happen?
-checkYourAnswers.fireOrFlood.reasonableExcuse = Fire or flood
-checkYourAnswers.lossOfStaff.reasonableExcuse = Loss of staff essential to the VAT process
 checkYourAnswers.lossOfStaff.whenPersonBecameUnavailable = When did the person become unavailable?
```

Comment:
I believe there shouldn't be a difference between the heading and CYA page row, so you should be able to use the headingAndTitle for loss of staff as well

Reply:
in ref to: checkYourAnswers.lossOfStaff.whenPersonBecameUnavailable

Reply:
There seems to be a difference between heading and CYA page row in case of loss of staff(unlike other reasons)

![image](https://user-images.githubusercontent.com/86231805/161026196-cd9d3e8c-7b8c-4348-af17-8ae4cf23b0d4.png)

![image](https://user-images.githubusercontent.com/86231805/161026330-661e66e6-bae6-43aa-af57-a2ed4e91efe8.png)